test/topology_custom/test_raft_recovery_basic.py

```diff
@@ -52,6 +53,7 @@ async def test_raft_recovery_basic(manager: ManagerClient, random_tables: Random
     await asyncio.gather(*(wait_until_upgrade_finishes(cql, h, time.time() + 60) for h in hosts))
 
     logging.info("Upgrade finished. Creating a new table")
+    random_tables = RandomTables(request.node.name, manager, "ks", 1)
```

Comment:
The `random_tables` fixture seems to use `unique_name` function to generate a name for the keyspace. AFAIU clusters can be recycled in some circumstances, i.e. a cluster created by one test can be reused by another. If two tests use the same keyspace then the leftovers from the former test run may interfere with the latter.

I suggest using `unique_name` to generate the keyspace name for the sake of peace of mind (also need to be changed in other places within the same commit).

Reply:
All these tests mark the cluster dirty because of topology changes, so not using a unique keyspace name should be fine. But I agree that using `unique_name` looks better and is less confusing, so I did it anyway.